packages/loot-core/src/shared/util.test.ts

```diff
@@ -24,6 +26,15 @@ describe('utility functions', () => {
     expect(looselyParseAmount('3,4500000')).toBe(3.45);
     expect(looselyParseAmount('3,45000000')).toBe(3.45);
     expect(looselyParseAmount('3,450000000')).toBe(3.45);
+    expect(looselyParseAmount("3'456.78")).toBe(3456.78);
```

Comment:
Im not sure what to do about this failing lint.  Should we just disable lint for these lines?

Reply:
Yeah I agree. Made a PR [here](https://github.com/actualbudget/actual/pull/3114) to disable it for all test files, let me know if you think it's better to disable it selectively though